javascript/node/selenium-webdriver/testing/index.js

```diff
@@ -540,6 +547,11 @@ function getTestHook(name) {
 }
 
 function locate(fileLike) {
+  if (!runfiles) {
+    // eslint-disable-next-line n/no-process-exit
+    process.exit(1)
```

Comment:
If we're not running in Bazel, we should try the `fs.existsSync` path, and if that fails then return an error. We don't want to exit the process.

The mental model I'm thinking of it:

1. In all cases, check the local disk
2. If we're running in bazel, also check bazel's runfiles

Reply:
HI Simon,
Updated as suggested!